File: user-service/src/main/java/com/gallery/userservice/domain/UserID.java
package com.gallery.userservice.domain;

public class UserID {
    private int id;

    public UserID() { this.id = 0; }
    public UserID(int id) { this.id = id; }
    public UserID(UserID other) { this.id = other.id; }

    public int getId() { return id; }
    public void setId(int id) { this.id = id; }
}

File: app/UserService/Domain/UserID.cs
namespace UserService.Domain
{
    public class UserID
    {
        private int _userId;

        public UserID()
        {
            this._userId = 0;
        }

        public UserID(int userId)
        {
            this._userId = userId;
        }

        public UserID(UserID userId)
        {
            this._userId = userId._userId;
        }

        public int Id
        {
            get { return this._userId; }
            set { this._userId = value; }
        }
    }
}